docs/src/manual/variables-and-scoping.md

```diff
@@ -1,517 +1,395 @@
-arguments called ``x`` without the two ``x``'s referring to the same
-thing. Similarly there are many other cases where different blocks of
-code can use the same name without referring to the same thing. The
-rules for when the same variable name does or doesn't refer to the
-same thing are called scope rules; this section spells them out in
-detail.
-
-Certain constructs in the language introduce *scope blocks*, which are
-regions of code that are eligible to be the scope of some set of
-variables. The scope of a variable cannot be an arbitrary set of
-source lines; instead, it will always line up with one of these
-blocks.  There are two main types of scopes in Julia, *global scope*
-and *local scope*, the latter can be nested.  The constructs
-introducing scope blocks are:
-
-.. _man-scope-table:
-
-+--------------------------------+----------------------------------------------------------------------------------+
-| Scope name                     | block/construct introducing this kind of scope                                   |
-+================================+==================================================================================+
-| :ref:`global <man-global>`     | module, baremodule, at interactive prompt (REPL)                                 |
-+--------------------------------+------------------------------+---------------------------------------------------+
-| :ref:`local <man-local-scope>` | :ref:`soft <man-soft-scope>` | for, while, comprehensions,                       |
-|                                |                              | try-catch-finally, let                            |
-|                                +------------------------------+---------------------------------------------------+
-|                                | :ref:`hard <man-hard-scope>` | functions (either syntax, anonymous & do-blocks), |
-|                                |                              | type, immutable, macro                            |
-+--------------------------------+------------------------------+---------------------------------------------------+
-
-Notably missing from this table are :ref:`begin blocks
-<man-compound-expressions>` and :ref:`if blocks
-<man-conditional-evaluation>`, which do *not* introduce new scope
-blocks.  All three types of scopes follow somewhat different rules
-which will be explained below as well as some extra rules for
-certain blocks.
-
-Julia uses `lexical scoping <https://en.wikipedia.org/wiki/Scope_%28computer_science%29#Lexical_scoping_vs._dynamic_scoping>`_,
-meaning that a function's scope does not inherit from its caller's
-scope, but from the scope in which the function was defined.
-For example, in the following code the ``x`` inside ``foo`` refers
-to the ``x`` in the global scope of its module ``Bar``::
-
-    module Bar
-    x = 1
-    foo() = x
-    end
-
-and not a ``x`` in the scope where ``foo`` is used::
+# [Scope of Variables](@id scope-of-variables)
```

Comment:
`jlcon` is being used for all code blocks containing Julia REPLs since plain `julia` ends up highlighting results incorrectly sometimes, such as 

```julia
julia> x -> x
(::#1) (generic function with 1 method)
```

`jlcon` is used in the pygments highlighter to correctly highlighting Julia REPL blocks, so there's precedence for that naming convention. We're using highlight.js for highlighting, which I hope we can add an equivalent `jlcon` highlighter to eventually.

What were the other formatting issues you encountered?